ledger/participant-integration-api/src/main/scala/platform/store/dao/events/AnormParamsMapper.scala

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.platform.store.dao.events
+
+import java.sql.{PreparedStatement, Timestamp}
+import java.time.Instant
+
+import anorm.ToStatement
+
+private[events] object AnormParamsMapper {
```

Comment:
We already have an object `com.daml.platform.store.Conversions` containing conversion implicits for Anorm. Is there a specific reason you didn't add these implicits to that one?

Reply:
There is not. I found one of implicits in EventsTablePostgresql, so I assumed that we don't have such object.

Reply:
Moved them to Conversions